package.json

```diff
@@ -121,11 +121,11 @@
     "connected-react-router": "^6.9.0",
     "history": "^4.10.0",
     "immutable": "4.0.0-rc.10",
-    "lattice": "~0.90.0",
+    "lattice": "^0.90.0",
```

Comment:
I use `~` for our lattice dependencies in `peerDependencies` because minor versions contain breaking changes. For example, going from `v0.89.2` to `v0.90.0` requires extra work. With `^`, will a newer version automatically be installed during `npm i`?

Reply:
that's what i'm expecting this change to do.